raw-data/fotmob-leagues/get_fotmob_leagues.R

```diff
@@ -9,28 +8,90 @@ library(tidyr)
 library(janitor)
 library(here)
 
-page <- 'https://www.fotmob.com/' %>% 
+page <- "https://www.fotmob.com/" %>% 
```

Comment:
use double quotation marks everywhere since that's been the standard you've used. (as a contributor, i should try to maintain your practices as much as possible, where it makes sense.)

Reply:
Thanks heaps! I don't know that it was a conscious decision on my part, but appreciated!